db/migrations/versions/0dc6f757f011_.py

```diff
@@ -0,0 +1,49 @@
+"""empty message
+
+Revision ID: 0dc6f757f011
+Revises: 13d601bfaee1
+Create Date: 2022-11-12 08:13:12.206935
+
+"""
+import sqlalchemy as sa
+import sqlalchemy_utils  # noqa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = "0dc6f757f011"
+down_revision = "13d601bfaee1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("account", schema=None) as batch_op:
+        role = postgresql.ENUM(
+            "ADMIN",
+            "LEAD_ASSESSOR",
+            "ASSESSOR",
+            "COMMENTER",
+            "APPLICANT",
+            name="role",
+        )
+        role.create(op.get_bind())
+
+        batch_op.add_column(
+            sa.Column("role", role, server_default="APPLICANT", nullable=False)
```

Comment:
Are we sure this will work if there is existing data in the database without this field set?

Reply:
psql output on docker runner postgres db before and after migration:

Reply:
Migration output:

INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade 13d601bfaee1 -> 0dc6f757f011, empty message